universal-login-sdk/lib/core/utils/errors.ts

```diff
@@ -67,10 +67,10 @@ export class ValidationFailed extends SDKError {
   }
 }
 
-export class InvalidAddress extends ValidationFailed {
+export class InvalidAddressOrEnsName extends ValidationFailed {
   constructor(address: string) {
-    super(`Address ${address} is not valid`, 'InvalidAddress');
-    Object.setPrototypeOf(this, InvalidAddress.prototype);
+    super(`${address} is not valid`, 'InvalidAddressOrEnsName');
```

Comment:
Consider rename address variable (recipient maybe, but i am not sure)

Reply:
or `addressOrEnsName`

Reply:
Done